src/main/java/com/mixpanel/android/viewcrawler/ViewVisitor.java

```diff
@@ -38,7 +38,7 @@
         public void OnEvent(View host, String eventName, boolean debounce);
```

Comment:
I've done it. But why do you want to make it a static class in ViewVisitor? I think it's unique to LayoutUpdateVisitor.